plugins/destination/clickhouse/queries/tables.go

```diff
@@ -19,7 +19,7 @@ func sortKeys(table *schema.Table) []string {
 func CreateTable(table *schema.Table, cluster string, engine *Engine) string {
 	normalized := normalizeTable(table)
 	strBuilder := strings.Builder{}
-	strBuilder.WriteString("CREATE TABLE ")
+	strBuilder.WriteString("CREATE TABLE IF NOT EXISTS")
```

Comment:
Context: [migrate code](https://github.com/cloudquery/cloudquery/blob/main/plugins/destination/clickhouse/client/migrate.go#L115-L140).

This shouldn't be required as the migrate mode should drop the table.
I suspect, the real issue here is missing `ON CLUSTER` in the `DROP` statement. See #9377

Reply:
AGREE！ This is better